lib/ui/painting.dart

```diff
@@ -1215,6 +1289,23 @@ class Image extends NativeFieldWrapperClass2 {
   /// The number of image pixels along the image's vertical axis.
   int get height native 'Image_height';
 
+  /// Converts the [Image] object into a byte array.
+  ///
+  /// The [format] is encoding format to be used.
+  ///
+  /// Returns a future which complete with the binary image data (e.g a PNG or JPEG binary data) or
+  /// an error if encoding fails.
+  Future<ByteData> toByteData({EncodingFormat format: const EncodingFormat.jpeg()}) {
```

Comment:
https://github.com/google/skia/blob/master/src/image/SkImage.cpp#L117

Skia seem to be using lossless PNG format as the default when encoding format is not specified.
Here, are we using JPEG as the default ?
Or should we pass null and use Skia's default ?

Reply:
Pass `const EncodingFormat.png()` as the format to get lossless PNG encoding.